.github/workflows/docs-pr.yml

```diff
@@ -44,7 +42,21 @@ jobs:
           yarn-docs: 'true'
       - name: ➕ Add `bin` to GITHUB_PATH
         run: echo "$(pwd)/bin" >> $GITHUB_PATH
+      - name: 🧶 Yarn install
+        if: steps.expo-caches.outputs.yarn-docs-hit != 'true'
+        run: yarn install --frozen-lockfile
+      - name: 🧪 Run Docs tests
+        run: yarn test
+      - name: 🚨 Lint Docs website code
+        run: yarn lint --max-warnings 0
+      - name: 💬 Lint Docs website content
+        run: yarn lint-prose
+      - name: ⚠️ Danger check
+        run: yarn danger ci
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       - name: 🪣 Set up docs preview bucket
+        if: contains(github.event.pull_request.labels.*.name, 'preview')
```

Comment:
Honestly I think we should rename the label to clarify it's a preview of the docs, e.g. `docs preview`. Right now it's a bit confusing.

Reply:
I think it's a good idea, let's do this separately.